rest_framework/schemas/openapi.py

```diff
@@ -471,13 +470,15 @@ def map_field(self, field):
         if isinstance(field, serializers.FloatField):
             content = {
                 'type': 'number',
+                'format': 'float'
```

Comment:
these formats could be helpfull, I think. Probably the only thing to be merged in DRF.